test/Base_Tests/src/Data/Decimal_Spec.enso

```diff
@@ -336,6 +347,16 @@ add_specs suite_builder =
                     (d <= d) . should_be_true
                     (d >= d) . should_be_true
 
+                    Problems.expect_only_warning Loss_Of_Numeric_Precision (d <= v)
+                    Problems.expect_only_warning Loss_Of_Numeric_Precision (d >= v)
+                    Problems.expect_only_warning Loss_Of_Numeric_Precision (v <= d)
+                    Problems.expect_only_warning Loss_Of_Numeric_Precision (v >= d)
+
+                    Problems.expect_only_warning Loss_Of_Numeric_Precision (d < d)
+                    Problems.expect_only_warning Loss_Of_Numeric_Precision (d > d)
+                    Problems.expect_only_warning Loss_Of_Numeric_Precision (d <= d)
+                    Problems.expect_only_warning Loss_Of_Numeric_Precision (d >= d)
```

Comment:
Why do we have these warnings here? I don't think I understand. I'm comparing a Decimal with Decimal - why would that give me a warning???

Reply:
Yes, this is quite wrong and I missed it because of other problems with the test and with comparison. I marked this pending and it will be completed in #10163.